ndk/src/events/index.ts

```diff
@@ -292,12 +292,6 @@ export class NDKEvent extends EventEmitter {
             this.author = await signer.user();
         }
 
-        await this.generateTags();
```

Comment:
Both of these are called as part of the `toNostrEvent()` function call on line 301